cedar-policy/src/api/err.rs

```diff
@@ -1161,3 +1170,53 @@ pub mod request_validation_errors {
     #[diagnostic(transparent)]
     pub struct TypeOfContextError(#[from] cedar_policy_core::entities::json::GetSchemaTypeError);
 }
+
+/// An error generated by entity slicing.
+/// See [`FailedAnalysisError`] for details on the fragment
+/// of Cedar handled by entity slicing.
+// CAUTION: this type is publicly exported in `cedar-policy`.
+// Don't make fields `pub`, don't make breaking changes, and use caution
+// when adding public methods.
```

Comment:
No need for the warning here since this definition is directly in `cedar-policy`. Of course it doesn't hurt though 🤷‍♀️

Reply:
I'll keep it since I keep moving it around (: